COCOAPODS.md

```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
+
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("pod_dependency") {
+              source = url("https://github.com/Kotlin/kotlin-with-cocoapods-sample/raw/cocoapods-zip/cocoapodSourcesZip.zip", flatten = true)
+           }
+
+       }
+   }
+   ```
+   
+   </div>
+
+3. Re-import the project.
+
+> To work correctly with Xcode, you should specify the path to the Podspec in your Podfile.
+> For example:
+>
+> <div class="sample" markdown="1" theme="idea" data-highlight-only>
+>
+> ```ruby
+> target 'ios-app' do
+>     # ... other pod depedencies ...
+>    pod 'podspecWithFilesExample', :path => '../kotlin-libr/build/cocoapods/externalSources/url/podspecWithFilesExample'
+> end
+> ```
+> 
+> </div>
+>
+{:.note} 
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Add a dependency on a Pod library from a custom Spec repository
+
+You can add dependencies on a Pod library from a custom Spec repository with `pod()` and `specRepos` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the HTTP address to the custom Spec repository using the `url()` inside the `specRepos` block.
+
+2. Specify the name of a Pod library in the `pod()` function.
+
+3. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           specRepos {
+              url("https://github.com/Kotlin/kotlin-cocoapods-spec.git")
+           }
+           pod("example")
+
+       }
+   }
+   ```
+
+   </div>
+
+4. Re-import the project.
+
+> To work correctly with Xcode, you should specify the location of specs at the beginning of your Podfile.
+> For example, `source 'https://github.com/Kotlin/kotlin-cocoapods-spec.git'`
+{:.note} 
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Add a dependency on a Pod library with custom C interoperability options
```

Comment:
Again. about cinterop | C Interop | C interoperability options. In other docs, blog posts, and what's new, we used cinterop.